ompi/mca/osc/rdma/osc_rdma_lock.h

```diff
@@ -227,8 +225,6 @@ static inline int ompi_osc_rdma_btl_cswap (ompi_osc_rdma_module_t *module, uint8
             ret = OMPI_SUCCESS;
         }
 
-        /* need to release here because ompi_osc_rdma_atomic_complete was not called */
-        OBJ_RELEASE(pending_op);
```

Comment:
I'm not sure about this. OBJ_NEW creates an object with a ref count of 1. The following OBJ_RETAIN put this refcount to 2. I don't see 2 OBJ_RELEASE in the error path here, so I think it is safe to assume we are leaking memory.

Reply:
Same as the other, I removed this change. Thanks for catching - it's no longer needed with the switch to active message put.